package.json

```diff
@@ -15,11 +15,14 @@
     "apm": "./bin/apm"
   },
   "scripts": {
-    "build": "node script/check-version.js && grunt",
-    "prepare": "grunt prepare",
+    "check-version": "node script/check-version.js",
+    "clean": "shx rm -rf lib bin/node_darwin_x64 && shx mkdir -p lib",
+    "lint": "coffeelint src spec",
+    "coffee": "coffee --compile --output lib src",
+    "build": "npm run clean && npm run coffee",
```

Comment:
The original `build` script did this stuff but also ran the linter at the end.

I think this should be `npm run clean && npm run coffee && npm run lint`

Reply:
I don't want to follow what was being used blindly. I don't see a reason to lint the code after coffeescript has compiled the code.

Reply:
Okay, that sounds basically reasonable.

But doesn't this mean we don't run the linter on pull requests/in CI anymore? I would still want to do that to make sure our code is formatted cleanly.

Reply:
Linting is part of test script.

Reply:
I didn't see that, thanks for pointing it out.